src/migrations/20190521104500_add_hold_until_index.js

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright: Ambrosus Inc.
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+// eslint-disable-next-line import/prefer-default-export
+export const up = async (db, config, logger) => {
+  await db.collection('bundle_metadata').createIndex({'repository.holdUntil': -1});
+
+  logger.info(`Added index to bundle_metadata.repository.holdUntil`);
+};
```

Comment:
Why wouldn't we check every bundle with blockchain data and assign proper `holdUntil`?
(Also in this scenario we could determine which bundles should be erased)

Reply:
We store holdUntil in metadata already, so it shouldn't be necessary.